docs/sdk/src/reference_docs/transaction_extensions.rs

```diff
@@ -0,0 +1,57 @@
+//! Transaction extensions are, briefly, a means for different chains to extend the "basic"
```

Comment:
Now is a good time to make sure this page is up-to-date and in good shape. I would start by asking: does this contain all the information necessary to learn about these tx extensions? It seems to me that there is a lot of valuable information that is only mentioned in the PR doc and not anywhere for posterity. 

I know @jsdw had interest in this in the `SignedExtension` era, but now @georgepisaltu it is your call to decide what to do with this page. @jsdw might still be up for collaboration ofc.

Reply:
Opened #3591 and I'm open to assisting and reviewing a contribution from @jsdw :grin: